abstar/utils/indels.py

```diff
@@ -96,8 +96,8 @@ def json_formatted(self):
         j = {'in_frame': 'yes' if self.in_frame else 'no',
              'length': self.length,
              'sequence': self.sequence,
-             'position': self.imgt_position,
-             'codon': self.imgt_codon}
+             'position': str(self.imgt_position),
```

Comment:
Coercing as string, otherwise some values are integers while others are floating points for these IMGT positions. For example position `112.1`. These positions do not have any numerical significance, hence they can instead be treated as string values.